pkg/tanka/parallel.go

```diff
@@ -30,6 +30,11 @@ func parallelLoadEnvironments(envs []*v1alpha1.Environment, opts parallelOpts) (
 		opts.Parallelism = defaultParallelism
 	}
 
+	if opts.Parallelism > len(envs) {
+		log.Info().Int("parallelism", opts.Parallelism).Int("envs", len(envs)).Msg("Reducing parallelism to number of environments")
```

Comment:
What do you think of?

```suggestion
		log.Info().Int("parallelism", opts.Parallelism).Int("envs", len(envs)).Msg("Reducing parallelism to match number of environments")
```